client/src/components/Login.jsx

```diff
@@ -11,7 +11,7 @@ class Login extends React.Component {
     };
 
     this.handleChange = this.handleChange.bind(this);
-    this.saveHost = this.saveHost.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
```

Comment:
I changed this implementation a bit in my latest PR so this might cause some merge conflicts depending on what we merge first!